technical-reports/index.html

```diff
@@ -72,7 +72,7 @@ <h1>Modules</h1>
           <a href="./format/">Format</a>
         </li>
         <li><a href="./color/">Color</a></li>
-        <li>Animations (coming soon)</li>
```

Comment:
🤔 I have no idea what happens to animations but in the future I’d love to resume this exploration (I’m no Val Head but I’m interested and have some knowledge here)